mat_nn_openmp.hpp

```diff
@@ -54,9 +65,12 @@ double su3_mat_nn(std::vector<site> &a, std::vector<su3_matrix> &b, std::vector<
   }
 
   for (int iters=0; iters<iterations+warmups; ++iters) {
+
+#ifndef ALIGNED_WORK
```

Comment:
We should just delete these ALIGNED_WORK pragmas throughout this file, we don't need it anymore.